doc/user/ospf6d.rst

```diff
@@ -83,6 +83,15 @@ OSPF6 router
    This configuration setting MUST be consistent across all routers
    within the OSPF domain.
 
+.. index:: maximum-paths (1-64)
+.. clicmd:: maximum-paths (1-64)
+
+.. index:: no maximum-paths [(1-64)]
```

Comment:
No "no" in the index lines, please. you can just have one clicmd line, with "[no]"

Reply:
Done